ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -210,6 +210,12 @@ def exp(x, name=None):
     return ivy.exp(x)
 
 
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
```

Comment:
Just a small correction, based on the documentation it seems that the function also supports `float16`. Other than that, great!